### PR TITLE
Error on routing screen - ResourceStep missing dateArchive

### PR DESCRIPTION
**Pull request by veggiematts**
Fix for issue #190

### CODE REVIEW

**Comment by jeffnm** [comment]
@veggiematts thanks again for making this PR. Can you make the requested change?

File: resources/install/protected/install.sql [hunk 1]
```diff
@@ -475,6 +475,7 @@ CREATE TABLE  `ResourceStep` (
   `stepID` int(11) default NULL,
   `stepStartDate` date default NULL,
   `stepEndDate` date default NULL,
+  `archivingDate` DATETIME NULL,
```
Comment (veggiematts) [inline resources/install/protected/install.sql]: It is already in the install.sql file. Am I missing something here?

**Comment by veggiematts** [comment]
It is already in the install.sql file. Am I missing something here ?

**Comment by jeffnm** [comment]
Maybe I don't understand. The Insert statements in the update file are adding rows to several tables, but I don't see that in the install file. The starting "Any" values for those tables should be generated at install as well, or they should not be added during the update. Otherwise some will have them and others not. Right? 

I'll add line comments when I get to a computer to point out the ones I mean. The GitHub app doesn't seem to have that ability.

**Comment by veggiematts** [comment]
Oh, I see. They are here as well. See upcoming review lines.

**Comment by jeffnm** [comment]
Thanks! That clears up my confusion. I'll merge.

**Pull request merged**